dynamodump/dynamodump.py

```diff
@@ -1121,12 +1120,11 @@ def main():
         "-r",
         "--region",
         help="AWS region to use, e.g. 'us-west-1'. "
-        "Can use AWS_DEFAULT_REGION for local testing.  Use '"
-        + LOCAL_REGION
-        + "' for local DynamoDB testing",
+        "Can use any region for local testing",
     )
     parser.add_argument(
-        "--host", help="Host of local DynamoDB [required only for local]"
+        "--host",
+        help="Host of local DynamoDB. This parameter triggers going to a local environment. [required only for local]",
```

Comment:
Could you please reword the second part of this sentence to `Host of local DynamoDB. This parameter initialises dynamodump for local DynamoDB testing [required only for local]`

Reply:
Done